### PR TITLE
Remove webpack-dev-server path and add --skip-open option

### PR DESCRIPTION
**Pull request by michaelgilley**
I'd also suggest adding the `--bg` flag to the wiki advanced usage page.

### CODE REVIEW

**Comment by alansouzati** [comment]
looks good, but can we choose another name for the `bg` option? maybe `--skip-open` ?

I'm trying to avoid short names for options. 

Can you add docs in README for the new option?

**Comment by michaelgilley** [comment]
Makes sense to me!

File: README.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Grommet-toolbox augments gulp object with these additional tasks:
 * **gulp scsslint**: uses `scssAssets` option to lint your Sass code. `scsslint` option must also be set to true.
 * **gulp jslint**: uses `jsAssets` and `testPaths` options to lint your JavaScript code.
 * **gulp dev**: starts a webpack dev server with hot reloading. See options for example configuration.
+  * `--config`: Set the path of the config file to use.
+  * `--skip-preprocess`: Skips preprocess tasks.
+  * `--skip-open`: Skips opening dev server url in a browser.
```
Comment (alansouzati) [inline README.md]: thanks for adding docs for the other options as well 👏

**Comment by alansouzati** [comment]
thanks for your contribution.

**Pull request merged**